requests_toolbelt/adapters/appengine.py

```diff
@@ -44,6 +49,57 @@ def __init__(self, validate_certificate=True, *args, **kwargs):
         super(AppEngineAdapter, self).__init__(self, *args, **kwargs)
 
     def init_poolmanager(self, connections, maxsize, block=False):
-        self.poolmanager = gaecontrib.AppEngineManager(
-            validate_certificate=self._validate_certificate
-        )
+        self.poolmanager = _AEPoolManager(self._validate_certificate)
+
+
+class _AEPoolManager(object):
+    """Implements urllib3's PoolManager API."""
```

Comment:
I'm unsure what you want me to document here...(or down below, either)

Before, init_poolmanager was creating an AppEngineManager, which was not actually a pool manager, and didn't actually work at all. Now init_poolmanager is creating an AppEnginePoolManager, and the poolmanager's connection_from_url constructs an AppEngineConnection, which seems to be straightforward and obvious. The classes are really only needed to implement the APIs that requests library expects of the AppEngine adapters, and the docstrings already say that...

Maybe I'm too deep in this to see what needs documenting here, but if you give me an example of what you were thinking docstring-wise, I can fill it out...